src/backend/commands/functioncmds.c

```diff
@@ -409,15 +409,56 @@ interpret_function_parameter_list(ParseState *pstate,
 
 		if (fp->defexpr)
 		{
-			Node	   *def;
+			Node *def;
+			Node *paramDft;
 
 			if (!isinput)
 				ereport(ERROR,
 						(errcode(ERRCODE_INVALID_FUNCTION_DEFINITION),
 						 errmsg("only input parameters can have default values")));
 
-			def = transformExpr(pstate, fp->defexpr,
+			/*
+			 * Intercept unquoted string defaults in parameter default declarations in 
+			 * T-SQL CREATE PROCEDURE/CREATE FUNCTION.
+			 * These arrive here as nodetype=T_ColumnRef. Temporarily change
+			 * the node type to T_TSQL_UnquotedString, which is picked up and 
+			 * handled in transformExprRecurse().
+			 */
+			paramDft = fp->defexpr;
+			if (sql_dialect == SQL_DIALECT_TSQL && 
```

Comment:
It'll fit babel's design principle to put those into a hook, and define a new hook into postgres code

Reply:
moved into hook functions